patches/chrome-BUILD.gn.patch

```diff
@@ -52,15 +52,19 @@ index 5670687f18f166da366afba61f7bbc3ecdf48acb..373e32e1b3b8c8153920fa807719e10d
      }
    }
  
-@@ -595,6 +602,7 @@ if (is_win) {
+@@ -595,9 +602,11 @@ if (is_win) {
        "--scm=0",
      ]
    }
 +  copy("brave_helper_plist") { deps = [ "//brave:brave_helper_plist" ] sources = [ "$root_gen_dir/brave/brave_helper_plist_tweaked.plist" ] outputs = [ "$root_gen_dir/chrome/{{source_file_part}}" ] }
  
    compile_entitlements("entitlements") {
      entitlements_templates = [ "app/app-entitlements.plist" ]
-@@ -618,7 +626,7 @@ if (is_win) {
++    entitlements_templates += [ "//brave/app/app-entitlements-brave.plist" ]
```

Comment:
seems like this should be gated on is_official_build

Reply:
Updating